content/applications/marketing/surveys/overview/create.rst

```diff
@@ -1,88 +1,228 @@
+live session survey. Reward participants for quick answers by selecting the checkbox labeled
+:guilabel:`Reward quick answers`. By checking it, attendees will get more points if they answer
+quickly.
+
+Description tab
+===============
+
+Back on the main survey template page is the :guilabel:`Description` tab, where a custom
+description of the survey can be added. This is displayed beneath the title on the survey's
+homepage, which is on the front end of the website made through the Odoo :guilabel:`Website` app.
 
 Test and share the survey
 =========================
 
-| Once your Survey is ready, *Test* it to avoid *Sharing* it with potential errors. As answers get
-  collected, click on *Answers* to access all the details of the respondent and his answers.
-| By default, a filter *Except Test Entries* is applied to keep the list of entries clean with only
-  real participants.
+Once the survey is created and saved, run a test to check for possible errors before finally
+sending it out to the participants by clicking :guilabel:`Test` in the upper left corner of the
+survey template page.
 
-.. image:: create/user_feedback.png
-   :align: center
-   :alt: View list of the participations of a survey in Odoo Surveys
+When activated, Odoo redirects the page to a test version of the survey on the front end of the
+website. This page displays how the survey will look to participants. Proceed to run through the
+survey, like a normal participant, to check for errors.
 
-| Another way to access detailed answers is by going to :menuselection:`Participations -->
-  Participations`.
-| Click on *See results* to be redirected to an analytical page.
+To return to the survey template form in the backend, simply click the :guilabel:`This is a test
+survey. Edit Survey` link in the blue banner along the top of the page. Once Odoo redirects the
+page to the survey template in the backend, make any further changes, as needed, before officially
+sending the survey out to participants.
 
-.. image:: create/analytical_page.png
-   :align: center
-   :alt: View of an analytical page for Odoo Surveys
+When the survey is ready to be shared with the audience, click the :guilabel:`Start Survey` button
+in the upper-left corner of the survey template form. Then, click :guilabel:`Share`.
+
+In the pop-up window, add the survey recipients in the :guilabel:`Recipients` field (for existing
+contacts in the Odoo database) or the :guilabel:`Additional emails` field (for contacts that do not
+want to be listed in the Odoo database). Finally, click :guilabel:`Send`.
+
+As answers are received, check them by clicking the :guilabel:`Answers` smart button on the survey
+template form, or the :guilabel:`See Results` button in the upper left corner. To end the survey,
+click the :guilabel:`Close` button on the survey template form.
 
 .. seealso::
-   - :doc:`scoring`
-   - :doc:`time_random`
+    - :doc:`scoring`
+    - :doc:`time_random`
```

Comment:
Why? This list was [correctly indented](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#be-consistent-with-indentation) before your change.

The same applies to the other lists on the page.